src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -0,0 +1,813 @@
+
+            legacySurveySchema = new UploadSchema();
+            legacySurveySchema.setSchemaId("legacy-survey");
+            legacySurveySchema.setRevision(1L);
+            legacySurveySchema.setName("Legacy (RK/AC) Survey");
+            legacySurveySchema.setSchemaType(UploadSchemaType.IOS_SURVEY);
+            legacySurveySchema.setFieldDefinitions(Lists.newArrayList(def1,def2));
+            uploadSchemasApi.createUploadSchema(legacySurveySchema).execute();
+        }
+
+        ddbSynapseMetaTables.putItem(new Item().withPrimaryKey("tableName", studyId + "-appVersion"));
+        ddbSynapseMetaTables.putItem(new Item().withPrimaryKey("tableName", studyId + "-status"));
+        ddbSynapseTables.putItem(new Item().withPrimaryKey("schemaKey", studyId + "-legacy-survey-v1"));
+
+        uploadId = testUpload("./legacy-survey-encrypted");
+
+        // then create test synapse project and team
+        createSynapseProjectAndTeam();
+    }
+
+    @After
+    public void after() throws Exception {
+        ddbSynapseMetaTables.deleteItem("tableName", studyId + "-appVersion");
+        ddbSynapseMetaTables.deleteItem("tableName", studyId + "-status");
+        ddbSynapseTables.deleteItem("schemaKey", studyId + "-legacy-survey-v1");
+
+        if (studyId != null) {
+            admin.getClient(StudiesApi.class).deleteStudy(studyId, true).execute();
+        }
+        if (project != null) {
+            synapseClient.deleteEntityById(project.getId());
+        }
+        if (team != null) {
+            synapseClient.deleteTeam(team.getId());
+        }
+
+        ddbExportTimeTable.deleteItem("studyId", studyId);
+
+        if (StringUtils.isNotBlank(s3FileName)) {
+            s3Client.deleteObject(recordIdOverrideBucket, s3FileName);
+        }
+
+        admin.signOut();
+    }
+
+    @AfterClass
+    public static void deleteWorker() throws Exception {
+        if (worker != null) {
+            worker.signOutAndDeleteUser();
+        }
+    }
+
+    @AfterClass
+    public static void deleteDeveloper() throws Exception {
+        if (developer != null) {
+            developer.signOutAndDeleteUser();
+        }
+    }
+
+    @AfterClass
+    public static void deleteResearcher() throws Exception {
+        if (researcher != null) {
+            researcher.signOutAndDeleteUser();
+        }
+    }
+
+    @AfterClass
+    public static void deleteUser() throws Exception {
+        if (user != null) {
+            user.signOutAndDeleteUser();
+        }
+    }
+
+    @Test
+    public void testInstant() throws Exception {
+        DateTime dateTimeBeforeExport = DateTime.now();
+        Long epochBeforeExport = dateTimeBeforeExport.getMillis(); // use for later verification
+        LOG.info("Time before request instant exporting: " + dateTimeBeforeExport.toString());
+
+        TimeUnit.MINUTES.sleep(1); // wait for gray period
```

Comment:
since we can't change logic in bridge pf side (I assume), so each time we send instant export request to server, we need to wait 1 min at least (that's the design decision right?) we can't change datetime now for server side so probably this 1 min is inevitable. one workaround is to execute each test parallel -- we still need to wait but following tests can be executed at one time.